arkouda/numeric.py

```diff
@@ -2141,3 +2123,68 @@ def clip(
     if hi is not None:
         pda1 = where(pda1 > hi, hi, pda1)
     return pda1
+
+
+def median(pda):
+    """
+    Compute the median of a given array.  1d case only, for now.
+
```

Comment:
numpy can handle boolean.  How hard would this be to add?  Can it be included or should it be a separate ticket?
```
In [29]: np.median(np.array([True, True, False]))
Out[29]: 1.0

```

Reply:
ak.sort doesn't take bool, so we'd have to convert, but I don't see any reason not to do it.